src/components/metadata/MetadataItem.vue

```diff
@@ -25,31 +29,62 @@
 </template>
 
 <script setup lang="ts">
-import { computed } from 'vue';
-import MetadataLink from '@/components/metadata/MetadataLink.vue';
-import MetadataValue from '@/components/metadata/MetadataValue.vue';
+import { computed } from "vue";
+import MetadataLink from "@/components/metadata/MetadataLink.vue";
+import MetadataValue from "@/components/metadata/MetadataValue.vue";
+import CopyCitation from "@/components/metadata/CopyCitation.vue";
+import { useConfigStore } from "@/stores/config";
 
 const props = defineProps<{
-  item: Metadata,
+  item: Metadata;
 }>();
 
-const label = computed<string>(() => props.item?.key || 'other');
+const label = computed<string>(() => props.item?.key || "other");
 const childItems = computed<Metadata>(() => props.item?.metadata || []);
+const configStore = useConfigStore();
 
 function isLink(): boolean {
-  const regex = /(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#/%?=~_|!:,.;]*[-A-Z0-9+&@#/%=~_|])/ig;
-  const matches = (typeof props.item?.key === 'string') ? props.item?.key?.match(regex) : null;
+  const regex =
+    /(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#/%?=~_|!:,.;]*[-A-Z0-9+&@#/%=~_|])/gi;
+  const matches =
+    typeof props.item?.key === "string" ? props.item?.key?.match(regex) : null;
 
   return matches !== null;
 }
 
+function getMetadataView(panels) {
+  let panelMetadata = panels.filter(
+    (panel) => panel.label.toLowerCase() === "metadata"
+  );
+  let metadataView;
+  if (panelMetadata.length > 0) {
+    // when metadata is one separate panel
+    metadataView = panelMetadata[0].views[0];
+  } else {
+    // when there is one panel containing content and metadata views
+    const panelContainingMetadata = panels.filter((panel) =>
+      panel.label.toLowerCase().includes("metadata")
```

Comment:
This line breaks TIDO when testing with EUPT config. The reason is that you try to parse a label value. Please find a better way. We should never parse user given values in generic products!